src/sagemaker_xgboost_container/data_utils.py

```diff
@@ -25,8 +26,8 @@
 LIBSVM = 'libsvm'
 
 
-INVALID_CONTENT_TYPE_ERROR = "{invalid_content_type} is not an accepted ContentType:" \
-                             " 'csv', 'libsvm', 'text/csv', 'text/libsvm', 'text/x-libsvm'."
```

Comment:
Minor: These accepted ContentTypes should be put in a list so that we aren't copy/pasting raw string values. No need to submit a review for this change.

Reply:
Do you mean just for this error message, or for all references to these content types? I went ahead and created a list just for the error message, but it might be worthwhile to migrate everything into a dictionary (though this would be a separate issue to deal with).

Reply:
There should be a list that can be referenced throughout the codebase so that we aren't checking against a copy pasted list repeatedly. This isn't just for the error messages but throughout the repo.

Reply:
That would mean changing the way we're referencing these content types on a large scale. We should definitely do this, but I'm not sure this PR is the right place to make this change. What do you think?

Reply:
That's fair. Can we make the list for the error messages for now, since they refer to the same exact list? No need to resubmit for approval.